### ISSUE TITLE
proc_macro does not have a constructor for c string literals

### ISSUE THREAD

**Issue by MaxVerevkin** [rust-lang/rust#118560]
C string literals have been stabilized (#117472) but there is no way to create them in proc macros.

**Comment by novafacing** [comment]
This should be resolved by #119750 / #119651. C String literals can now be constructed like:

```rust
let lit = proc_macro::Literal::c_string(c"a");
```

(or, equivalently)

```rust
let lit = proc_macro::Literal::c_string(&std::ffi::CString::new("a").unwrap());
```

**Comment by fmease** [comment]
Closing as superseded by libs-api tracking issue.
